routers/api/v1/repo/issue.go

```diff
@@ -130,6 +138,37 @@ func SearchIssues(ctx *context.APIContext) {
 		opts.Private = true
 		opts.AllLimited = true
 	}
+	if ctx.Form("owner") != "" {
+		owner, err := models.GetUserByName(ctx.Form("owner"))
+		if err != nil {
+			if models.IsErrUserNotExist(err) {
+				ctx.Error(http.StatusBadRequest, "Owner not found", err)
+			} else {
+				ctx.Error(http.StatusInternalServerError, "GetUserByName", err)
+			}
+			return
+		}
+		opts.OwnerID = owner.ID
+		opts.AllLimited = false
+		opts.AllPublic = false
+		opts.Collaborate = util.OptionalBoolFalse
+	}
+	if ctx.Form("team") != "" {
```

Comment:
So maybe move this to line 155 is better, then we could delete line 157-160

Reply:
But then we can't return an error message to the user when the operation is called with missing "owner" parameter when filtering on team? The "team" parameter would be silently ignored.